fintech-examples/fintech-impl/src/main/java/de/adorsys/opba/fintech/impl/tppclients/TppAisClient.java

```diff
@@ -7,7 +7,7 @@
 import javax.servlet.http.HttpServletRequest;
 import java.util.Optional;
 
-@FeignClient(url = "${tpp.url}", name = "tppAisClient")
+@FeignClient(url = "${tpp.url}", name = "tppAisClient", decode404 = true)
```

Comment:
We should use FeignDecoder:
https://stackoverflow.com/questions/38786207/netflix-feign-propagate-status-and-exception-through-microservices
and have HttpStatus 429 for too frequent consent access,
and HttpStatus 410 for generic error that indicates consent was corrupted

Reply:
Done